server/__init__.py

```diff
@@ -98,30 +98,33 @@ def register_routes_custom_dc(app):
   pass
 
 
-def register_routes_disasters(app):
-  # Install blueprints specific to Stanford DC
+def register_routes_disasters(app, include_sustainability):
```

Comment:
why have "include_sustainability" here? I feel register_xxx() should be a simple function. All the condition check should be outside and, if needed, have another register_xxx() to hold other related logic groups.

Reply:
if sustainability is different from disaster. then can use SHOW_SUSTAINABILITY flag in the env py files. We should try to remove the if/else checks here as much as possible..